### ISSUE TITLE
Activities triggered by Harmony remote not showing in Homebridge or Home.app

### ISSUE THREAD

**Issue by bverkron** [nicoduj/homebridge-harmony#32]
Since updating to the latest version (0.1.2) activities triggered via the Harmony remote no longer show in the Homebridge log and are not changing state in Home.app. I triggered several activities via the remote and none showed in either place.

Activities triggered in Home.app DO show in the Homebridge log and update their state correctly in Home.app

Getting some errors in the log too but it's not entirely clear when they are triggered. Seemingly when I triggered an activity in the Home.app but not every time.

```
0|homebridge  | ReferenceError: callback is not defined
0|homebridge  |     at Timeout._onTimeout (/usr/lib/node_modules/homebridge-harmony/index.js:425:47)
0|homebridge  |     at listOnTimeout (timers.js:324:15)
0|homebridge  |     at processTimers (timers.js:268:5)
```

Also nothing except sometimes this deprecation warning on startup or maybe when triggering with the remote.
```
0|homebridge  | (node:16430) [DEP0112] DeprecationWarning: Socket.prototype._handle is deprecated
```

**Comment by nicoduj** [comment]
Thank you for feedback I will dig into this asap

**Comment by nicoduj** [comment]
I checked quickly and the line of the error is not consistent. Could you please tell me : 
- your homebridge / node.js versions
- what your config.json file looks like
- give me a log in debug that reproduce the problem

**Comment by nicoduj** [comment]
ensure to check your refreshTimer option, since it was defaulted to 300s if not in the accepted range of 15s-600s. I moved back the range to 5-600s in latest version

**Comment by bverkron** [comment]
Possible but something doesn't fit. Previously and despite not changing the refresh setting in the configs (so should have defaulted) the response in the logs and Home.app seemed to be near instantaneous. Now despite setting `"refreshTimer" : 5` I get nothing in the Homebridge output and Home.app takes multiple minutes to update (and still nothing is shown in the Homebridge output).

![image](https://user-images.githubusercontent.com/19174503/51863313-88dc9480-22f5-11e9-9717-70c8411d9b75.png)

Earlier in this troubleshooting I was at least getting entries in the Homebridge output after triggering activities on the Home.app side. Now nothing.

**TL;DR**

Activity started on remote = nothing in Homebridge output. Several minutes to update in Home.app or never at all (have to restart Home.app to update the status).
Activity started in Home.app = Instant response but nothing shown in Homebridge output.

Current config.json
```
"platforms": [
    {
        "platform": "HarmonyHubWebSocket",
    	"name": "Harmony Hub (homebridge)",
    	"hubIP": "192.168.1.104",
	"showTurnOffActivity" : true,
        "refreshTimer" : 5
    }
```

This error appears sporadically. Sometimes there seems to be a loose correlation with start / stop of activities on remote or Home.app but not every time.
```
0|homebridge  | ReferenceError: callback is not defined
0|homebridge  |     at Timeout._onTimeout (/usr/lib/node_modules/homebridge-harmony/index.js:425:47)
0|homebridge  |     at listOnTimeout (timers.js:324:15)
0|homebridge  |     at processTimers (timers.js:268:5)
```

**Comment by bverkron** [comment]
I rolled back to 0.9.0 (could not remember which version I was on before, took a guess).

Home.app shows updated status much faster now and does not need to be killed / reopened to update. 

Homebridge still does not show anything in the output when switching activities from the remote side but did show output when stopping an activity from Home.app

Perhaps the activity refresh in Home.app is something in Homebridge-harmony while the lack of output is due to something in the Homebridge update.  hmmmm.

**Comment by nicoduj** [comment]
Hi, it is strange since the error you mention has been fixed in 0.1.3 ( I also had it before and it made homebridge crash that could explain the behavior)
I would also suggest to move the refreshTimer to 15s to see if it helps.
Your warning of deprecation is also strange what is your node.js version ? 
You can also try to switch back to only one accessory through the option publishActivitiesAsIndividualAccessories to false

I reopen the issue does anyone else have this behavior ?

**Comment by bverkron** [comment]
My priorities have shifted especially since it "works" with an older version. I will do my best to get the requested info and do additional testing and get back to you.

**Comment by nicoduj** [comment]
all right, I will close if nobody else has the problem in a while :)
